package/network/utils/uqmi/files/lib/netifd/proto/qmi.sh

```diff
@@ -68,6 +69,9 @@ proto_qmi_setup() {
 		return 1
 	}
 
+	[ "$mtu" = "" ] && mtu="1500"
+	/sbin/ip link set dev $ifname mtu $mtu
```

Comment:
You're changing the current behavior, you're effectively forcing MTU to 1500 if it's not set, I think, that it would make more sense to just run that `ip link set` command only in case that mtu config option is set.

Reply:
@yezz thanks for your reply.
Have changed the behavior so that the MTU is only set if this option is not empty.